### PR TITLE
P0 feature bug fixes

### PR DESCRIPTION
**Pull request by abhisheksahani-nova**

### CODE REVIEW

**Review by preethasri — APPROVED** [review]
good to merge

File: src/components/cards/verticalCard/VerticalCard.jsx [hunk 1]
```diff
@@ -21,14 +21,14 @@ function VerticalCard({ video }) {
   const [showPlaylistDropdown, setShowPlaylistDropdown] = useState(false);
 
   return (
-    <div>
+    <div onClick={() => addToHistory(video, token)}>
       <div className="videolib-verticalcard-resize">
         <div className="badge-container">
           <img
             className="card-img videolib-verticalcard-img-resize"
             src={img}
```
Comment (MayankKumar10) [inline src/components/cards/verticalCard/VerticalCard.jsx:30]: add alt tag

File: src/components/dropdown/playlistDropdown/playlistDropdown.css [hunk 2]
```diff
@@ -1,12 +1,26 @@
+.playlist-dropdown-container {
+  display: block;
+  position: fixed;
+  width: fit-content;
+  z-index: 1;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
```
Comment (MayankKumar10) [inline src/components/dropdown/playlistDropdown/playlistDropdown.css:9]: you don't need to use height 100% here

**Review by MayankKumar10 — APPROVED** [review]
Look into these Suggestions Everything else looks good to me you can merge it.

**Pull request merged**